src/lib/faktory-fun.ts

```diff
@@ -0,0 +1,5 @@
+import FaktorySDK from "@faktoryfun/core-sdk";
+
+export const sdkMainnet = new FaktorySDK({
+    network: 'mainnet'
+})
```

Comment:
Do we need a mainnet specific object here, or should we pass the ENV for network?

```suggestion
export const faktorySdk = (network: string) => new FaktorySDK({ network })
```